.github/ISSUE_TEMPLATE.md

```diff
@@ -2,21 +2,24 @@
 Thank you for reporting an issue.
 
 This issue tracker is for bugs and issues found within gitcoin.
-If you require more general support reach out to us at
+If it's general support you need, reach out to us at
```

Comment:
stay away from contractions. Also should this read as `its` not `it's` nor 'it is'? My grammar is poor. If the latter than do away with the contraction.

Reply:
My bad :P